code/modules/clothing/clothing.dm

```diff
@@ -140,6 +143,70 @@
 		how_cool_are_your_threads += "</span>"
 		. += how_cool_are_your_threads.Join()
 
+	if(LAZYLEN(armor_list))
+		armor_list.Cut()
+	if(armor.bio)
+		armor_list += list("TOXIN" = armor.bio)
+	if(armor.bomb)
+		armor_list += list("EXPLOSIVE" = armor.bomb)
+	if(armor.bullet)
+		armor_list += list("BULLET" = armor.bullet)
+	if(armor.energy)
+		armor_list += list("ENERGY" = armor.energy)
+	if(armor.laser)
+		armor_list += list("LASER" = armor.laser)
+	if(armor.magic)
+		armor_list += list("MAGIC" = armor.magic)
+	if(armor.melee)
+		armor_list += list("MELEE" = armor.melee)
+	if(armor.rad)
+		armor_list += list("RADIATION" = armor.rad)
+
+	if(LAZYLEN(durability_list))
+		durability_list.Cut()
+	if(armor.fire)
+		durability_list += list("FIRE" = armor.fire)
+	if(armor.acid)
+		durability_list += list("ACID" = armor.acid)
+
+	if(LAZYLEN(armor_list) || LAZYLEN(durability_list))
+		. += "<span class='notice'>It has a <a href='?src=[REF(src)];list_armor=1'>tag</a> listing its protection classes.</span>"
+
+/obj/item/clothing/Topic(href, href_list)
+	. = ..()
+
+	if(href_list["list_armor"])
+		var/list/readout = list("<span class='notice'><b>PROTECTION CLASSES (I-V)</b>")
+		if(LAZYLEN(armor_list))
+			readout += "\n<b>ARMOR</b>"
+			for(var/dam_type in armor_list)
+				var/armor_amount = armor_list[dam_type]
+				readout += "\n[dam_type] [number_to_level(armor_amount)]" //e.g. BOMB IV
+		if(LAZYLEN(durability_list))
+			readout += "\n<b>DURABILITY</b>"
+			for(var/dam_type in durability_list)
+				var/durability_amount = durability_list[dam_type]
+				readout += "\n[dam_type] [number_to_level(durability_amount)]" //e.g. FIRE II
+		readout += "</span>"
+
+		to_chat(usr, "[readout.Join()]")
+
+/obj/item/clothing/proc/number_to_level(number)
+	switch (number)
+		if (1 to 20)
+			. = "I"
+		if (21 to 40)
+			. = "II"
+		if (41 to 60)
+			. = "III"
+		if (61 to 80)
+			. = "IV"
+		if (81 to 99)
+			. = "V"
+		if (100 to INFINITY)
+			. = "∞"
+	return .

```

Comment:
this doesn't need to be a proc, just edit durability amount after 188 to convert itself into level then just print out `[durability_amount]`.

Reply:
You should have that from 1-19, 20-29 and so on, because as it stands the Elite Hardsuit and Deathsquard Hardsuit are in the same category when they are nowhere remotely the same level of protection (the Deathsquad hardsuit is exactly 80 and is the highest of any armor)

Reply:
ok